spec/lib/myfinance/resources/entity_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'spec_helper'
+
+describe Myfinance::Resources::Entity do
+  describe '.find_all', vcr: true do
+    context 'when success' do
+      subject { described_class.find_all }
+
+      it 'returns a find_all of orders' do
+        expect(subject.class).to eq(Myfinance::Resources::EntityCollection)
+        expect(subject.collection.first.class).to eq(Myfinance::Resources::Entity)
+        expect(subject.collection.first.id).to eq(3798)
+        expect(subject.collection.first.name).to eq("Minhas Finanças")
+        expect(subject.collection.first.account_id).to eq(3613)
+        expect(subject.collection.first.charging_uuid).to be_nil
+        expect(subject.collection.first.created_at).to eq(DateTime.parse("2015-07-29T11:04:45-03:00"))
+        expect(subject.collection.first.default_in_menu).to be_truthy
+        expect(subject.collection.first.deleted_at).to be_nil
+        expect(subject.collection.first.federation_subscription_number).to eq("")
+        expect(subject.collection.first.imported_from_sync_at).to be_nil
+        expect(subject.collection.first.updated_at).to eq(DateTime.parse("2015-07-29T14:30:36-03:00"))
+        expect(subject.collection.count).to eq(1)
+      end
+    end
+
+    context 'when not found' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.